src/strategies/index.ts

```diff
@@ -395,7 +398,10 @@ const strategies = {
   'push-voting-power': pushVotingPower,
   'staked-psp-balance': stakedPSPBalance,
   'erc20-balance-of-contract-multiplier': erc20BalanceOfContractMultiplier,
-  agave
+  agave,
+  'snet-liquidity-providers': snetLiquidityProviders,
+  'snet-stakers': snetStakers,
+  'snet-farmers': snetFarmers
```

Comment:
Hi @Vivek205 Would be great if you can do separate PR for each of these strategies, will be easy to review/integrate

Reply:
Sure. Will update it.

Reply:
@ChaituVR 

raised a separate PR for snet-liquidity-providers
https://github.com/snapshot-labs/snapshot-strategies/pull/202

Reply:
https://github.com/snapshot-labs/snapshot-strategies/pull/203

https://github.com/snapshot-labs/snapshot-strategies/pull/204